### PR TITLE
fix(gitignore): remove package-lock

### LINKED ISSUE #8: Remove Package-lock file
In the recent PR a package-lock file was accidentally added.

### PR DESCRIPTION
**Pull request by felipebrunetti94**
- deleted package-lock file
- package-lock file to gitignore

Closes #8

### CODE REVIEW

**Comment by felipebrunetti94** [comment]
@negreirosleo

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@
 /.vs
 /node_modules/.bin
 /node_modules
+/package-lock
```
Comment (negreirosleo) [inline .gitignore]:
```suggestion
/package-lock.json
```

**Review by negreirosleo — APPROVED** [review]
LGTM

**Review by jrtechs — APPROVED** [review]
Looks great thanks @felipebrunetti94

**Pull request merged**